packages/imperative/src/rest/src/session/ConnectionPropsForSessCfg.ts

```diff
@@ -127,13 +127,24 @@ export class ConnectionPropsForSessCfg {
                     if (cmdArgs[argName] != null) { (sessCfgToUse as any)[override.propertyName] = cmdArgs[argName]; }
                     for (const prop of override.propertiesOverridden) {
                         // Make sure we do not prompt for the overridden property.
-                        if (!doNotPromptForValues.includes(prop)) { doNotPromptForValues.push(prop); }
+                        if (!doNotPromptForValues.includes(prop as any)) { doNotPromptForValues.push(prop as any); }
```

Comment:
If possible we should avoid using `as any` here. Tested locally and after removing `as any` was getting errors about type `string | number | symbol` not matching type `string`.

I think this can be fixed by restricting the type to `(keyof SessCfgType & string)[]` in 4 places (inspired by [StackOverflow](https://stackoverflow.com/a/54784203/5504760) 😋):
* `promptForValues` in this method
* `doNotPromptForValues` in this method
* `propertiesOverridden` on the `IOverridePromptConnProps` interface
* `propsToPromptFor` on the `IOverridePromptConnProps` interface